docs/src/tut/mkt/index.md

```diff
@@ -1003,6 +1004,7 @@ That's all that is needed for the declaration of variables.
 Proceed to the frontend file.
 
 #### {#tut-mkt-Interaction-advance-define} Define More Variables
+
 The variables that have just been added to the backend file will now be defined or given their functionalities here.
```

Comment:
will now be defined or given their functionalities here. <-- kind of repetitious. I would suggest only using one.